src/test/java/org/openrewrite/okhttp/MigrateToOkHttp4Test.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.okhttp;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.config.Environment;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.openrewrite.maven.Assertions.pomXml;
+
+class MigrateToOkHttp4Test implements RewriteTest {
+
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(Environment.builder()
+          .scanRuntimeClasspath("org.openrewrite.okhttp")
+          .build()
+          .activateRecipes("org.openrewrite.okhttp.UpgradeOkHttp4"));
```

Comment:
If there's only a single resource involved we don't have to scan the classpath; that helps keep tests fast when more changes are added.
```suggestion
        spec.recipeFromResource("/META-INF/rewrite/okhttp-4.yml", "org.openrewrite.okhttp.UpgradeOkHttp4"));
```